topics/statistics/tutorials/classification_machinelearning/tutorial.md

```diff
@@ -31,10 +34,31 @@ recordings:
   youtube_id: Gz2OdRPS2Nk
   speakers:
   - anuprulez
+- youtube_id: TODO
+  length: 28M
+  galaxy_version: " 24.1.2.dev0"
+  date: '2024-08-29'
+  speakers:
+  - anuprulez
+  captioners:
+  - anuprulez
```

Comment:
```suggestion
```

Reply:
@teresa-m @anuprulez What is the plan for the lecture videos?

Ideally we would add the slides to the GTN, and then add associate the lecture video with that. We do have a script somewhere that can convert powerpoint to GTN slide format, to use as a starting point (you will likely have to do some tweaking afterwards to make it look good but it should hopefully make it a lot less work to add your slides) what do you think?